configure

```diff
@@ -11461,157 +11427,6 @@ rm -f core conftest.err conftest.$ac_objext \
 pgac_args="$pgac_args with_xml_prefix with_xml_exec_prefix enable_xmltest"
 CFLAGS="$CFLAGS $XML_CPPFLAGS"
 
-# libhdfs3
```

Comment:
We do not build libhdfs3 separately now so we do not need to check the dependency.